src/models/converters/transactions/mod.rs

```diff
@@ -211,6 +212,34 @@ impl MultisigTransaction {
             .flatten()
             .unwrap_or(0)
     }
+
+    fn is_cancellation(&self) -> bool {
```

Comment:
Should we add a follow up issue for unit testing this method?

Reply:
Yeah, I will do that, it would be nice to have this deployed now and write the tests througout the day :)

https://github.com/gnosis/safe-client-gateway/issues/264